crates/hir-def/src/import_map.rs

```diff
@@ -446,65 +341,44 @@ pub fn search_dependencies(
 
     let mut stream = op.union();
 
-    let mut all_indexed_values = FxHashSet::default();
-    while let Some((_, indexed_values)) = stream.next() {
-        all_indexed_values.extend(indexed_values.iter().copied());
-    }
```

Comment:
I've seen #7912, but I couldn't reproduce the duplication and I don't really think it's possible due to how `fst` works. Let me know if I'm missing anything.